client/src/common/components/FormDialog.jsx

```diff
@@ -39,17 +42,33 @@ class FormDialog extends Component {
 
   handleDescriptionChange = description => this.setState({ description });
 
-  handleNameChange = name => this.setState({ name });
+  handleNameChange = name => this.setState({ name }, this.validateName);
+
+  validateName = callback => {
+    const { name } = this.state;
+
+    const errorMessageId = validateWith(value =>
+      validator.isLength(value, { min: 1, max: 32 })
```

Comment:
Please display "Field is too long" for messages that exceed the max length - currently it would display the "Field is required" message for a text of 35 characters.